src/EnergyPlus/ChilledCeilingPanelSimple.cc

```diff
@@ -401,7 +401,7 @@ void GetCoolingPanelInput(EnergyPlusData &state)
             ThisCP.CoolingCapMethod = DataSizing::CapacityPerFloorArea;
             if (!state.dataIPShortCut->lNumericFieldBlanks(5)) {
                 ThisCP.ScaledCoolingCapacity = state.dataIPShortCut->rNumericArgs(5);
-                if (ThisCP.CoolingCapMethod <= 0.0) {
+                if (ThisCP.ScaledCoolingCapacity <= 0.0) {
```

Comment:
correct variable, how in the world did you find this?

Reply:
@rraustad Thanks! @rraustad. @Myoldmopar left a comment in the original issue saying based on context it might be some kind of sizing variable. I looked at the context and thought it should be this one. One small question I have is the "context" actually seems to have use `< 0.0` rather than `<=0.0` for a couple of other places. Does this matter here--probably a little bit if there are frequent occurrences that the capacity is indeed zero?

Reply:
rNumericArgs(5) should be a real number >= 0 or an error is raised. So if a user wants 0 capacity, the IDD and code allow that. Given the current state of code (i.e., allowing 0 capacity) I don't see a problem.

    N5 , \field Cooling Design Capacity Per Floor Area
       \type real
       \units W/m2
       \minimum 0.0
       \note Enter the cooling design capacity per total floor area of cooled zones served by the unit.
       \note Required field when the cooling design capacity method field is CapacityPerFloorArea.

Reply:
@rraustad Thanks for the explanation! Based on this, actually this seems to have confirmed that here it should use the strict `< 0.0` rather than `<=0.0`, as the revised `if` block is for giving error messages: 
```
                ThisCP.ScaledCoolingCapacity = state.dataIPShortCut->rNumericArgs(5);
                if (ThisCP.ScaledCoolingCapacity <= 0.0) {
                    ShowSevereError(state, cCMO_CoolingPanel_Simple + " = " + ThisCP.EquipID);
                    ShowContinueError(state, "Input for " + state.dataIPShortCut->cAlphaFieldNames(5) + " = " + state.dataIPShortCut->cAlphaArgs(5));
                    ShowContinueError(
                        state, format("Illegal {} = {:.7T}", state.dataIPShortCut->cNumericFieldNames(5), state.dataIPShortCut->rNumericArgs(5)));
                    ErrorsFound = true;

```

Reply:
That sounds correct. I missed that.

Reply:
Thanks! I've pushed another commit to change this.